stock_location_orderpoint/tests/test_location_orderpoint.py

```diff
@@ -0,0 +1,296 @@
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from datetime import datetime
+
+from psycopg2.errors import UniqueViolation
```

Comment:
@mt-software-de As psycopg2 2.7.7 is in the requirements of Odoo for python < 3.8, it is better to use:

`from psycopg2 import IntegrityError`

That writing is compliant with both versions

Reply:
Yes this is better.